<launch>
	<arg name="namespace_arg" default="/xiroi" />
	
	<group ns="$(arg namespace_arg)">

		<arg name="joystickDevice"  default="/dev/input/js0"/>
		<node respawn="true" pkg="joy" type="joy_node" name="joystick">
		    <param name="dev" value="$(arg joystickDevice)"/>
		    <param name="deadzone" value="0.05"/>
		</node>
		<!-- TELEOPERATION -->
		<node respawn="true" pkg="xiroi" type="logitech_joy.py" name="logitech_joy" output="screen" />
		<node respawn="true" pkg="xiroi" type="teleoperation.py" name="teleoperation"/>
		<!-- <node respawn="true" pkg="xiroi" type="arduino_node.py" output="screen" name="arduino_node"/> -->

	</group>
</launch>